CHANGES.md

```diff
@@ -3,7 +3,7 @@
 Dependencies
 - Update to bytes `0.5` and amino_rs `0.5`.
 - Tokens for amino_rs are now fully non-conflicting with prost. Allowing both to be used together 
-
+- Made RPC type values optional for full compatibility with tendermint-go: `abci_info`, `abci_query` [#120]
```

Comment:
Maybe add a version number (0.32.x ?).

Reply:
I thought those were universally optional in tendermint-go. I'll check.